fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/impl/datagen/FabricDataGenHelper.java

```diff
@@ -42,6 +42,11 @@
 public final class FabricDataGenHelper {
 	private static final Logger LOGGER = LoggerFactory.getLogger(FabricDataGenHelper.class);
 
+	/**
+	 * When enabled the dedicated server startup will be hijacked to run the data generators and then quit.
+	 */
+	public static final boolean ENABLED = System.getProperty("fabric-api.datagen") != null;
```

Comment:
This property doesn't exist on NeoForge. If we're going to add the field back for mods that require internals access, we should at least set it to something realiable. An analogous system prop should exist in neo iirc.